quasar_entities/src/event.rs

```diff
@@ -0,0 +1,119 @@
+//! `SeaORM` Entity. Generated by sea-orm-codegen 0.12.3
```

Comment:
I think we can skip this line. Our files are heavily edited, even if created with a tool initially.

Reply:
https://github.com/eigerco/quasar/pull/43/commits/97627f34fa9fa1f052a0071ffd4fe7b4431b6519